### PR TITLE
Make input-columns configurable

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
- Make existing input-columns editable with on a modal
- Add settings of visibility on the modal

## Why?
To improve usability

### Before
スクリーンショット 2021-10-14 14 37 31
スクリーンショット 2021-10-14 14 37 48

### After
スクリーンショット 2021-10-14 14 38 09
スクリーンショット 2021-10-14 14 38 16

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/173#" title="Author self-approved">d-hayashi</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by WatanabeToshimitsu — CHANGES_REQUESTED** [review]
あざます
めっちゃ使いやすくなってると思います！！

例の如く細かいとこ指摘させてもらいました

File: src/components/organisms/InputConfigList.tsx
Comment (WatanabeToshimitsu) [reply src/components/organisms/InputConfigList.tsx:155]: 試してみたら，確かに更新される奴まで動いちゃってたので，これは無視してください

File: src/components/molecules/InputConfigListItem.tsx
Comment (d-hayashi) [reply src/components/molecules/InputConfigListItem.tsx:47]: ご指摘あざます！
修正しておきます

**Pull request merged**
